README.md

```diff
@@ -16,6 +16,8 @@ Get the API key [here](https://dashboard.fiddlecube.ai/api-key).
 
 ### Usage
 
+## Generate the Dataset
```

Comment:
let's remove the results from the example below

Reply:
Done